libraries/xcode.rb

```diff
@@ -1,5 +1,5 @@
 module Xcode
```

Comment:
If we're going to update name the namespace, let's match the convention of the rest of the helper modules and put `MacOS` as the top level and `XcodeHelper` as the second, where the name that comes before `Helper` is the name of the resource.

Reply:
Done.